src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/general/_variables.scss

```diff
@@ -4,15 +4,17 @@
 $color-text-normal: #444;
 $color-text-introvert: lighten($color-text-normal, 15%);
 $color-text-extrovert: darken($color-text-normal, 10%);
+$color-text-darkest: black;
```

Comment:
I don't get why we need yet another dark color. We already have `$color-text-normal`.